### ISSUE TITLE
Object removal task evaluation code

### ISSUE THREAD

**Issue by harsh-sengar** [vt-vl-lab/FGVC#53]
Thankyou for the great code. 

"Results in the object removal setting, however, are difficult to compare and evaluate due to the lack of ground truth content behind the masked object. For this reason, we introduce a further synthetic object mask inpainting task. Specifically, we take a collection of free-form object masks and randomly pair them with other videos, pretending there is an object occluding the scene"

Can you please explain it with concrete example how you are evaluating your code on object removal task. Are you using some other dataset for which ground truth is available. I wanted to compare results for my project and I am not able to get the evaluation(PSNR, SSIM, LPIPS) and training code. Can you point me to some public library for this, it would be very useful.

**Comment by huynth1801** [comment]
Hello. I also have the same question. Have you had solution for it ?

**Comment by cyrala** [comment]
> 

Hi! I also cannot get the evaluation(PSNR, SSIM, LPIPS) method as original paper, have you finally get the result similar to table 1?
